<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>314</height>
    </rect>
   </property>
   <property name="title">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="btn_create">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>230</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>215</y>
      <width>274</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>107</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Offset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>140</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Random</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_offset">
    <property name="geometry">
     <rect>
      <x>228</x>
      <y>104</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>76</y>
      <width>274</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_path">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>30</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Select path</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_loops">
    <property name="geometry">
     <rect>
      <x>69</x>
      <y>104</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>140</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Radius A</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_radius">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>137</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>107</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Loops</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_random">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>137</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_selectPath">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>20</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>path</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_Create">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>301</y>
      <width>271</width>
      <height>6</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>168</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Radius B</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spin_radius_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>168</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QCheckBox" name="btn_makeMesh">
    <property name="geometry">
     <rect>
      <x>187</x>
      <y>168</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Create Mesh</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="btn_reverse">
    <property name="geometry">
     <rect>
      <x>187</x>
      <y>194</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Reverse</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
